Exosite.h

```diff
@@ -76,6 +76,7 @@ class Exosite
     boolean DataRx;
     boolean RxLoop;
     char c;
+    unsigned long requestTimeout;
```

Comment:
Is this new variable used in long polling only? If so it would be good to change it to `longPollTimeout` or something more descriptive.